cmd/compute-domain-daemon/computedomain.go

```diff
@@ -34,6 +35,8 @@ const (
 	informerResyncPeriod = 10 * time.Minute
 )
 
+type IPSet = map[string]struct{}
```

Comment:
Consider using this set implementation:
https://pkg.go.dev/k8s.io/apimachinery/pkg/util/sets

Reply:
`maps.Equal()` on `map[string]struct{}` is I think good enough for now. For more sophisticated set operations, I keep `k8s.io/apimachinery/pkg/util/sets` in mind.

Reply:
(done)